src/Stratis.FederationGatewayD/start-gateway-testnet.ps1

```diff
@@ -0,0 +1,98 @@
+﻿
+#############################
+#    UPDATE THESE VALUES    #
+#############################
+$git_repos_path = "C:\Users\matthieu\source\repos"
+$root_datadir = "C:\Users\matthieu\AppData\Roaming\StratisNode"
+$path_to_federationgatewayd = "$git_repos_path\FederatedSidechains\src\Stratis.FederationGatewayD"
+$path_to_mining_key_dat_file = "$git_repos_path\secrets\federationKey.dat"
+$multisig_public_key = "03b824a9500f17c9fe7a4e3bb660e38b97b66ed3c78749146f2f31c06569cf905c"
+$mining_public_key = "0248de019680c6f18e434547c8c9d48965b656b8e5e70c5a5564cfb1270db79a11"
+$nickname = "matthieu"
+######################################
+#    UPDATE THIS BUT DO NOT SHARE    #
+######################################
+$multisig_mnemonic = "please change that to the keywords generated when using federation setup tool"
```

Comment:
Also here you can add "please add the path to the federationKey.dat file generated when using federation setup tool." and put it in $path_to_mining_key_dat_file